specification/subscription/resource-manager/readme.md

```diff
@@ -23,6 +23,13 @@ To see additional help and options, run:
 ``` yaml
 directive:
   - suppress: R2059
+  - suppress: PutResponseCodes
+    from: subscriptions.json
+    where: $.paths["/subscriptions/{subscriptionId}/providers/Microsoft.Subscription/changeTenantRequest/default"].put
+    reason: Not supposed to return 201 as the response code for the below API since existing api with new version change, got exceptions from ARM reviewer.
+  - suppress: PutRequestResponseSchemeArm
```

Comment:
Have you got an exception from ARM reviewer as well?

Reply:
Yes @gary-x-li , got an Exception from Roopesh to suppress with the reason provided.